lib/Runtime/Library/MathLibrary.cpp

```diff
@@ -147,7 +147,7 @@ namespace Js
     {
         double result;
 
-#if defined(_M_IX86)
+#if defined(_M_IX86) && defined(_WIN32) // TODO: xplat support
```

Comment:
That probably isn't necessary. This is really for perf.  Functionally, this is not necessary.
If the xplat CRT implementation already support SSE2, then we get most of the perf already.
It just avoid the is SSE2 support check at the beginning of these functions.

I'd rather not maintain our copy of these trig functions.